vhdbuilder/packer/install-dependencies.sh

```diff
@@ -490,8 +490,7 @@ cacheKubePackageFromPrivateUrl() {
   # use azcopy with MSI instead of curl to download packages
   getAzCopyCurrentPath
 
-  export AZCOPY_AUTO_LOGIN_TYPE="MSI"
-  export AZCOPY_MSI_RESOURCE_STRING="$LINUX_MSI_RESOURCE_IDS"
+  ./azcopy login --login-type=MSI
```

Comment:
@cameronmeissner  I though that performing a login was forcing the use of the system assigned MSI ?  the export would have been a way to force the default config for azcopy

Reply:
No system assigned MSI in 1es pool vms. We only have user assigned msi.

Reply:
@djsly @cameronmeissner  I tested `.\azcopy.exe login --login-type=MSI; .\azcopy.exe copy $URL $Dest` in  `vhdbuilder/packer/configure-windows-vhd.ps1` in this PR and it works in Windows VHD build VM
https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=95010287&view=logs&j=ceeb8f07-abdb-53f4-60d8-80121461daa9&t=9925fa4f-3645-534b-dffc-a869b4c81936&l=479
```
    azure-arm: 2024-06-03T07:45:56.2436378+00:00: Caching private packages on VHD
    azure-arm: 2024-06-03T07:45:56.2592588+00:00: Downloading a private package to c:\akse-cache\private-packages\v1.29.2-hotfix.20240530-1int.zip
    azure-arm: 2024-06-03T07:45:56.2592588+00:00: Creating temp dir for tools of building vhd
    azure-arm:
    azure-arm:
    azure-arm:     Directory: C:\
    azure-arm:
    azure-arm:
    azure-arm: Mode                LastWriteTime         Length Name
    azure-arm: ----                -------------         ------ ----
    azure-arm: d-----         6/3/2024   7:45 AM                akstemp
    azure-arm: 2024-06-03T07:45:56.3061336+00:00: Downloading azcopy
    azure-arm: 2024-06-03T07:46:00.0322330+00:00: Downloaded azcopy
    azure-arm: 2024-06-03T07:46:00.0322330+00:00: Moving azcopy to c:\akstemp
    azure-arm: 2024-06-03T07:46:00.0478640+00:00: Moved azcopy to c:\akstemp
    azure-arm: 2024-06-03T07:46:00.0478640+00:00: C:\akstemp
    azure-arm:
    azure-arm:
    azure-arm:     Directory: C:\akstemp
    azure-arm:
    azure-arm:
    azure-arm: Mode                LastWriteTime         Length Name
    azure-arm: ----                -------------         ------ ----
    azure-arm: d-----         6/3/2024   7:45 AM                tmp
    azure-arm: -a----        5/22/2024   9:07 PM       33735192 azcopy.exe
    azure-arm: -a----         6/3/2024   7:45 AM       16541185 azcopy.zip
    azure-arm: INFO: Login with identity succeeded.
    azure-arm: INFO: Scanning...
    azure-arm: INFO: Autologin not specified.
    azure-arm: INFO: Authenticating to source using Azure AD
    azure-arm: INFO: Any empty folders will not be processed, because source and/or destination doesn't have full folder support
    azure-arm:
    azure-arm: Job 48292dbc-d7db-6f4f-4c45-27d1b9f57b9a has started
    azure-arm: Log file is located at: c:\akstemp\azcopy\48292dbc-d7db-6f4f-4c45-27d1b9f57b9a.log
    azure-arm:
    azure-arm:
    azure-arm: 100.0 %, 1 Done, 0 Failed, 0 Pending, 0 Skipped, 1 Total, 2-sec Throughput (Mb/s): 264.5239
    azure-arm:
    azure-arm:
    azure-arm: Job 48292dbc-d7db-6f4f-4c45-27d1b9f57b9a summary
    azure-arm: Elapsed Time (Minutes): 0.0336
    azure-arm: Number of File Transfers: 1
    azure-arm: Number of Folder Property Transfers: 0
    azure-arm: Number of Symlink Transfers: 0
    azure-arm: Total Number of Transfers: 1
    azure-arm: Number of File Transfers Completed: 1
    azure-arm: Number of Folder Transfers Completed: 0
    azure-arm: Number of File Transfers Failed: 0
    azure-arm: Number of Folder Transfers Failed: 0
    azure-arm: Number of File Transfers Skipped: 0
    azure-arm: Number of Folder Transfers Skipped: 0
    azure-arm: Total Number of Bytes Transferred: 66628308
    azure-arm: Final Job Status: Completed

```